Jenkinsfile

```diff
@@ -196,8 +196,8 @@ jobWrapper {
                 buildAndroidArm64Debug  : doAndroidBuildInDocker('arm64-v8a', 'Debug'),
                 buildAndroidTestsArmeabi: doAndroidBuildInDocker('armeabi-v7a', 'Debug', TestAction.Build),
                 buildEmscripten         : doBuildEmscripten('Debug'),
-                threadSanitizer         : doCheckSanity(buildOptions + [enableSync: true, sanitizeMode: 'thread']),
-                addressSanitizer        : doCheckSanity(buildOptions + [enableSync: true, sanitizeMode: 'address']),
+                threadSanitizer         : doCheckSanity(buildOptions + [buildType: 'RelWithDebInfo', enableSync: true, sanitizeMode: 'thread']),
+                addressSanitizer        : doCheckSanity(buildOptions + [buildType: 'RelWithDebInfo', enableSync: true, sanitizeMode: 'address']),
```

Comment:
I'd suggest removing these build types entirely since they are now covered by evergreen. Not blocking, but it could help ease us of relying on Jenkins.

Reply:
Yeah, that's what i was going to question also.. Are these build environments identical with evergreen so that they are redundant, right? There is nothing what we lose by removing these builders?

Reply:
We can remove them from Jenkins, yes.